Cargo.toml

```diff
@@ -13,6 +13,7 @@ etagere = "0.2.10"
 cosmic-text = "0.11"
 lru = { version = "0.12.1", default-features = false }
 rustc-hash = "1.1"
+once_cell = "1.19"
```

Comment:
I'd be ok with bumping the MSRV to use `OnceCell` on stable if you'd like

Reply:
Oh, I actually had no idea it was now a part of the `std`. Good to know!

Alright, let me bump that.